browser/resources/settings/brave_overrides/index.js

```diff
@@ -28,6 +28,7 @@ import './people_page.js'
 import './personalization_options.js'
 import './printing_page.js'
 import './privacy_page.js'
+import './reset_profile_dialog.js'
```

Comment:
This regression came from this?

Reply:
Essentially yes, looks like it was accidentally omitted in this commit: https://github.com/brave/brave-core/commit/052be99c4f6fa79611c6fc1bdeac89b9efd99e49